api/src/main/java/com/community/api/endpoint/serviceProvider/ServiceProviderEntity.java

```diff
@@ -35,9 +39,10 @@ public class ServiceProviderEntity  {
     private String last_name;
     //@TODO-countryCode to country_code for both customer and service provider
     private String country_code;
+
     private String father_name;
 
-    private Date date_of_birth;
+    private String date_of_birth;
 
     private String aadhaar_number;
```

Comment:
Use camelcase in varible naming and use annotation @jsonProperty for snake_casing in postman.
As in java camelcasing is used globally don't mix both casing.
![image](https://github.com/user-attachments/assets/317fb3fa-08ee-4f2d-b1b1-a12b2d538b2b)